Makefile

```diff
@@ -13,5 +13,18 @@ uninstall:
 	@echo "and do 'unalias rm' from all your terminal sessions"
 	@echo "Successfully removed $(PREFIX)/safe-rm"
 
+# the test target will not work properly if a file named test is ever created
+# in this directory. Since it has no prerequisites, test would always be considered
+# up to date and its recipe would not be executed. To avoid this problem you
+# can explicitly declare the target to be phony by making it a prerequisite of
+# the special target
+# TODO: is test directory really needed?
```

Comment:
I see that there is a `test`directory, but I didn't understand if such code is useful or not. I need to define the `.PHONY` [target rule](http://www.gnu.org/savannah-checkouts/gnu/make/manual/html_node/Phony-Targets.html) in order to do tests